cate/core/workspace.py

```diff
@@ -582,7 +582,7 @@ def run_op(self, op_name: str, op_kwargs: OpKwArgs, monitor=Monitor.NONE):
 
             with monitor.starting("Running operation '%s'" % op_name, 2):
                 self.workflow.invoke(context=self._new_context(), monitor=monitor.child(work=1))
-                op(monitor=monitor.child(work=1), **unpacked_op_kwargs)
+                return op(monitor=monitor.child(work=1), **unpacked_op_kwargs)
```

Comment:
Note we can only return values that are JSON-serializable. Idea: add kwarg `returns: bool = False` that indicates that we expect a JSON-serializable output rather than side effects (e.g. bringing up a plot, writing a file). Otherwise return `None` (void).